packages/protocol/contracts-0.8/common/linkedlists/LinkedList.sol

```diff
@@ -0,0 +1,167 @@
+pragma solidity >=0.5.13 <0.8.20;
```

Comment:
There are huge breaking changes between 0.5 and 0.8 - eg SafeMath doesn't exist in 0.8 and it will not compile

Reply:
Also we have 0.8 already in stCelo (which was audited so we might want to consider to move it form there)

Reply:
The submodule [version](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/b53c43242fc9c0e435b66178c3847c4a1b417cc1/contracts/utils/math) we are using still has the safemath lib for >0.8.0.

I restricted the version range to only include 0.8.0 and above.